applications/NFT_Bridge_Protocol_for_NFT_Migration_and_Data_Exchange.md

```diff
@@ -28,133 +28,449 @@ In this proposal, we use the following terms and definitions:
+   </td>
+  </tr>
+  <tr>
+   <td>6
+   </td>
+   <td>Testing
+   </td>
+   <td>Tested on several EVM chains. Spent over a month of beta testing and bug fixing.
+   </td>
+   <td>Bridge <a href="https://github.com/Perpetual-Altruism-Ltd/myNFT-Bridge">Repository</a>
+<p>
+<a href="https://rinkeby.etherscan.io/txs?a=0xba5ad339c13446a9d00e380e9335be61d1ed4d4c&p=1">Rinkeby Test Contract</a>
+   </td>
+  </tr>
+</table>
+
+
+
+### Milestone 3 — Testing and Documentation, User Guides. (COMPLETED)
+* **Estimated Duration:** ~1 month
+* **FTE:** 3
+* **Work Days:** 21 days
+**Cost per work day:** £300
+* **Costs:** £6,300 (8464.24 USD/DAI)
+* **Deliverable:** We wrote the standard and documentation for cross-universe migration, which includes “necessary” and “optional” data for optimization. This is a summarized write up of both the previous milestones but written as specifications for implementations, including the predicate and postulate of each message and what they mean in terms of state on the source and destination universe.
+
+
+<table>
+  <tr>
+   <td>#
+   </td>
+   <td>Deliverable
+   </td>
+   <td>Specification
+   </td>
+   <td>Link
+   </td>
+  </tr>
+  <tr>
+   <td>1
+   </td>
+   <td>Migration Interface
+   </td>
+   <td>A standard interface written in Solidity including predicate and postulate for each function and events that would allow for either IOU or full migrations to be performed for the same asset, from the same bridge.
+   </td>
+   <td><a href="https://github.com/Perpetual-Altruism-Ltd/myNFT-Bridge/blob/master/Codebase/Bridge_Contracts/contracts/Bridge.sol">Bridge Interface</a>
+<p>
+<a href="https://github.com/Perpetual-Altruism-Ltd/myNFT-Bridge/blob/master/CONTRACT_ARCHI.md">Abi Json & solidity interface</a>
+   </td>
+  </tr>
+  <tr>
+   <td>2
+   </td>
+   <td>Established predicate and postulate on what are the allowed pair of source/destination worlds.
+   </td>
+   <td>Allowed pairs are designed in a decentralized trustless fashion.
+   </td>
+   <td><a href="https://docs.google.com/document/d/1c5Uor2By5igFWXimipcKhsWjTAG8OWrl9bSVWTPsi6U/edit#heading=h.gin6x1ljt7jq">See section seven</a>
+   </td>
+  </tr>
+  <tr>
+   <td>3
+   </td>
+   <td>Thorough user guide
+   </td>
+   <td>A documented step by step process that would allow a trusted relayer to perform efficient migrations of NFTs. 
+   </td>
+   <td><a href="https://github.com/Perpetual-Altruism-Ltd/myNFT-Bridge/blob/master/USERHOWTO.md">See the document </a>
+   </td>
+  </tr>
+  <tr>
+   <td>4
+   </td>
+   <td>License
+   </td>
+   <td>Licensed under the <a href="https://unlicense.org/">Unlicensed</a>.
+   </td>
+   <td><a href="https://github.com/Perpetual-Altruism-Ltd/myNFT-Bridge/blob/master/LICENSE">https://github.com/Perpetual-Altruism-Ltd/myNFT-Bridge/blob/master/LICENSE</a>
+   </td>
+  </tr>
+</table>
+
+
+
+## 
+**Main Deliverables 📦**
```

Comment:
Because this is what we expect people coming here to read : The beginning and the end of the grant. The bridge can be understood and used with only those deliverables, independently of the process that lead to them.

Reply:
Sounds good.